pages/hub/insights/index.tsx

```diff
@@ -42,69 +56,155 @@ const InsightsHub: WithPageLayout = () => {
       .then(() => {});
   }, [router, onboarded]);
 
+  useEffect(() => {
+    if (router.pathname === "/hub") {
+      router.push("/hub/insights");
+    }
+  }, [router]);
+
   return user && onboarded ? (
-    <div className="container flex flex-col w-full gap-4 py-2">
-      <div className="justify-between block py-2 sm:flex ">
-        <div>
-          <Title className="!text-2xl !leading-none !font-medium  mb-4" level={1}>
-            Insights
-          </Title>
-          <Text className="my-8">
-            Welcome to your Insights Hub! Here, you can set up pages to view all of your insights or other open source
-            insights in one place.
-          </Text>
-        </div>
-        <div className="flex items-center gap-3 mt-4">
-          {/* Search box temporarily hidden */}
-          <div className="hidden w-58">
-            <Search placeholder="Search repositories" className="max-w-full" name={"query"} />
+    <Tabs value={router.pathname.split("/")[2] as TabKey} onValueChange={onTabChange}>
```

Comment:
Why can't these be do distinct pages/components? I think the header should be common between the two in the layout, but they don't need to be tabs

Reply:
Thanks for this suggestion. I've implemented in the latest changes 👍